pkg/webhook/config.go

```diff
@@ -0,0 +1,74 @@
+package webhook
+
+import (
+	machinecfgv1 "github.com/openshift/api/machineconfiguration/v1"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
```

Comment:
interesting. Do we not have an ocp equiv?

Reply:
https://github.com/openshift/api/blob/f8aa75d189e1a9154d64ac74734f4d4158e0d6b8/install.go#L6